unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/DropdownItemColorSwap.cs

```diff
@@ -0,0 +1,33 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.SettingsPanelHUD.Controls
+{
+    public class DropdownItemColorSwap : MonoBehaviour
+    {
+        [SerializeField] private Toggle toggle;
+        [SerializeField] private TextMeshProUGUI targetText;
+        [SerializeField] private Image targetBackground;
+        [SerializeField] private Color textOn;
+        [SerializeField] private Color textOff;
+        [SerializeField] private Color bgOn;
+        [SerializeField] private Color bgOff;
+        private void OnEnable()
```

Comment:
A minor nitpick, but can you leave blank spaces around the methods?

Reply:
I keep missing them because rider adds a fake line between methods and the line before